### ISSUE TITLE
Rework logView

### ISSUE THREAD

**Issue by maplesteve** [maplesteve/homebridge-server#11]
- [x] Separate HTML code
- [x] page through the logfile
- [x] `tail -f` option
- [ ] `systemd`/`journalctl` option
- [x] log can be searched

**Comment by gismo141** [comment]
For journalctl there are a lot of parameters that can be used to filter or reverse the log-output.

Maybe we could add some kind of menubar for selectable buttons and a query textfield, what do you think?

**Comment by maplesteve** [comment]
Yes - journalctl is what I meant with 'systemd' (added it in the list).
I will set up a Docker with Ubuntu to play around with journalctl (macOS doesn't have it...).
Are you fine, when I open new issues like this in your repository? It's a bit pointless, if I track this stuff in my repo since it would have to go into yours at the end anyway...

**Comment by gismo141** [comment]
When you say Docker, I also tried to set it up there but somehow I couldn't add the homebridge to my iDevices. I used Alpine-Node which is very small and I have my dev-state as well in a [GitHub-Repo](https://github.com/gismo141/docker-homebridge).

But yes I use `journalctl` and `systemd` very often in conjunction as well.

Concerning where to place the Issues is a good point. I've added the keyword `v2.0.0` to the issues concerning not active problems but those we encountered during development. As well we loose the issue trace when I'm merging your pull-requests - so yes I think it would be better to set the issues in the main repo.

When committing you still can say `fixes #XYZ` and when I merge that pull later on it would fix the corresponding issue in the main repo as well.
